data/pandoc.lua

```diff
@@ -262,7 +264,7 @@ M.DefinitionList = M.Block:create_constructor(
 M.Div = M.Block:create_constructor(
   "Div",
   function(content, attr)
-    return {c = {attr or M.Attr(), content}}
+    return {c = {attr or M.Attr(), List:new(content)}}
```

Comment:
I'm curious why this change is needed?

Reply:
It's just for convenience, making it possible to write `div.content:find_if(fn)` rather than `List.find_if(div.content, fn)`. Users won't have to `require 'List'`. It doesn't appear to affect performance in a significant way.